pkgs/development/tools/tychus/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "tychus-${version}";
+  version = "0.6.3";
+
+  goPackagePath = "github.com/devlocker/tychus";
+  goDeps = ./deps.nix;
+  subPackages = [];
+
+  src = fetchFromGitHub {
+    owner = "devlocker";
+    repo = "tychus";
+    rev = "v${version}";
+    sha256 = "02ybxjsfga89gpg0k21zmykhhnpx1vy3ny8fcwj0qsg73i11alvw";
+  };
+
+
+  buildFlags = "--tags release";
+
+  meta = {
+    description = "Command line utility to live-reload your application.";
+    homepage = https://github.com/devlocker/tychus;
+    platforms = stdenv.lib.platforms.all;
```

Comment:
Please add a `license` field here. Although the package is licensed as `MIT` which is rather permissive we actually want to have packages with a documented license :)